sonar-batch/src/main/java/org/sonar/batch/cpd/CpdComponents.java

```diff
@@ -28,7 +30,7 @@
   private CpdComponents() {
   }
 
-  public static List all() {
+  public static List<? extends Object> all() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1452)